datadog_checks_dev/datadog_checks/dev/tooling/commands/env/start.py

```diff
@@ -30,10 +30,17 @@
         'config (agent5, agent6, etc.)'
     )
 )
+@click.option(
+    '--env-vars', '-v', default='',
```

Comment:
I'm open to suggestions for this naming, `-v` doesn't feel exactly right here but I didn't have an alternative.

Reply:
what about `-e`?

Reply:
I'm good with -e